zebra-network/src/peer/connection.rs

```diff
@@ -1659,12 +1659,12 @@ where
         self.erase_state_metrics();
 
         // Set the new state
-        metrics::increment_gauge!(
+        metrics::gauge!(
             "zebra.net.connection.state",
-            1.0,
             "command" => current_metrics_state.clone(),
             "addr" => self.metrics_label.clone(),
-        );
+        )
+        .set(1.0);
```

Comment:
`increment_gauge!(...)` needs to become `gauge!(...).increment(...)`

```suggestion
        .increment(1.0);
```